.vscode/tasks.json

```diff
@@ -0,0 +1,141 @@
+{
+  "version": "2.0.0",
+  "tasks": [
+    {
+      "label": "Aerie UI",
+      "type": "shell",
+      "command": "nvm use && npm run dev",
+      "detail": "Task to run just the UI for development.",
+      "problemMatcher": [
+        {
+          "background": {
+            "activeOnStart": true,
+            "beginsPattern": "  VITE",
+            "endsPattern": "➜  press h to show help"
+          },
+          "pattern": {
+            "regexp": ""
+          }
+        }
+      ],
+      "presentation": {
+        "group": "groupA"
+      }
+    },
+    {
+      "label": "Svelte Check",
+      "type": "shell",
+      "command": "nvm use && npm run check:watch",
+      "detail": "Task to run the svelte static analysis.",
+      "problemMatcher": [],
+      "presentation": {
+        "group": "groupA"
+      }
+    },
+    {
+      "label": "Unit Tests",
+      "type": "shell",
+      "command": "nvm use && npm run test",
+      "detail": "Task to run the unit tests with watch enabled.",
+      "problemMatcher": [],
+      "presentation": {
+        "group": "groupA"
+      }
+    },
+    {
+      "label": "Update Aerie",
+      "type": "shell",
+      "command": "cd ../aerie && git pull && ./gradlew assemble",
+      "detail": "Task to update your local Aerie repo and rebuild it.",
+      "problemMatcher": [],
+      "presentation": {
+        "group": "groupB"
+      }
+    },
+    {
+      "label": "Build Aerie",
+      "type": "shell",
+      "command": "cd ../aerie && ./gradlew assemble",
+      "detail": "Task to just rebuild Aerie without updating the repo.",
+      "problemMatcher": [],
+      "presentation": {
+        "group": "groupB"
+      }
+    },
+    {
+      "label": "Clean Aerie",
+      "type": "shell",
+      "command": "cd ../aerie && docker compose down && docker compose rm -f && docker compose down -v && docker compose down --rmi all && ./gradlew clean",
+      "detail": "Task to completely clean Aerie.",
+      "problemMatcher": [],
+      "presentation": {
+        "group": "groupB"
+      }
+    },
+    {
+      "label": "Aerie Containers",
+      "type": "shell",
+      "command": "cd ../aerie && docker compose up --build aerie_gateway aerie_merlin aerie_scheduler aerie_merlin_worker_1 aerie_merlin_worker_2 aerie_scheduler_worker_1 aerie_scheduler_worker_2 aerie_sequencing hasura postgres",
+      "detail": "Task to start up Aerie containers.",
+      "problemMatcher": [],
+      "presentation": {
+        "group": "groupB"
+      }
+    },
+    {
+      "label": "Dump Aerie DB",
```

Comment:
Consider "Wipe" or "Clear" instead of "Dump" - "Dump" to me is a little ambiguous and could mean "dump the contents to disk" instead of "clear contents of the db".